assets/javascripts/lib/discourse-markdown/bbcode-color.js.es6

```diff
@@ -21,7 +21,13 @@ function replaceFontBgColor(text) {
 }
 
 export function setup(helper) {
-  helper.whiteList(['font[color]']);
+  helper.whiteList({
+    custom(tag, name, value) {
+      if (tag === 'span' && name === 'style') {
+        return /^color:.*$/.exec(value);
```

Comment:
I am pretty sure this is a security hole cause you can then sneak in non color styles .* is way to permissive.